### PR TITLE
added materialize library

### PR DESCRIPTION
**Pull request by chrisjmckeown**
Added the materialize library with header, footer and columns, to get the map in the correct location

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -16,51 +16,88 @@
...
+                            <div class="card-content">
+                                <p></p>
+                            </div>
+                        </div>
+                        <div id="blurb-about-site" class="card">
+                            <div class="card-title">Blurb about Site</div>
+                            <div class="card-content">
+                                <p> The content written will apear in here this is the dive here it goes in</p>
+                            </div>
+                        </div>
```
Comment (smcheah) [inline index.html:79]: I think having the blurb card first is better (on top of the bandsintown list and scroll)

File: index.html [hunk 2]
```diff
@@ -16,51 +16,88 @@
...
+                    </div>
+                        <div class="col s12 m6 l6 map_canvas" id="map_canvas" >
+                    </div>
```
Comment (smoneo) [inline index.html]: how do you feel about putting the map in a card?
```suggestion
                    <div class="col s12 m6 l6">
                        <div class="card">
                            <div class="map_canvas" id="map_canvas"></div>
                        </div>
                    </div>
```

File: index.html (hunk 1 above)
Comment (smoneo) [inline index.html]: I think having the blurb card first is better (on top of the bandsintown list and scroll)
```suggestion
                        <div class="card" id="blurb-about-site">
                            <div class="card-title">Blurb about Site</div>
                            <div class="card-content">
                                <p> The content written will appear in here this is the dive here it goes in</p>
                            </div>
                        </div>
                        <div class="card">
                            <div class="card-title">The bands in Town</div>
                            <div class="card-content bands-in-town-list"></div>
                        </div>
                        <div class="card">
                            <div class="card-title">Possible carousel of bands</div>
                            <div class="card-content bands-in-town-scroll"></div>
                        </div>
```

**Review by smoneo — APPROVED** [review]
> **nice**

**Pull request merged**